app/views/shared/_secondary_navigation.html.erb

```diff
@@ -116,12 +116,16 @@
         </ul>
       <% elsif module_page? %>
         <ul class="nav navbar-nav navbar-right" style="vertical-align: bottom">
-          <li id="overview-nav-tab" class="<%= "active" if module_overview? %>">
-            <a href="<%= my_module_path(@my_module) %>" title="<%= t('nav2.modules.overview') %>">
-              <span class="hidden-sm hidden-md"><%= t('nav2.modules.overview') %></span>
-              <span class="hidden-xs hidden-lg glyphicon glyphicon-blackboard"></span>
-            </a>
-          </li>
+          <% if can_view_module(@my_module) %>
```

Comment:
So I suggest that you keep the previous code (the one that's removed). The `my_module_tab_li` function is meant for all tabs BUT overview (because it also saves the tab's name, in this case `overview`, into the `shown_tabs` JSONB column of `my_module`). The thing about overview is that it's ALWAYS shown, so I wouldn't trouble ourselves with making sure that each time you create `my_module`, you need to paste `overview` into its `shown_tabs` column, etc.